src/components/Button.tsx

```diff
@@ -2,11 +2,16 @@ import { ReactNode, MouseEventHandler } from "react";
 
 interface ButtonProps {
   children?: ReactNode;
+  className?: string;
   onClick?: MouseEventHandler;
 }
 
-const Button = ({ children, onClick }: ButtonProps) => {
-  return <button onClick={onClick}>{children}</button>;
+const Button = ({ children, className, onClick }: ButtonProps) => {
+  return (
+    <button className={`Button ${className ?? ""}`} onClick={onClick}>
```

Comment:
Standard naming is for class names to be in `kebab-case`, so lowercase `button` would be preferred here
```suggestion
    <button className={`button ${className ?? ""}`} onClick={onClick}>
```
Remember to also change the CSS!